A83-xds-gcp-authn-filter.md

```diff
@@ -375,13 +375,18 @@ credentials implementations in some languages.
 
 ## Implementation
 
-C-core implementation:
+#### C-core implementation:
```

Comment:
I don't think these need to be section headers.  It's fine for them to be just two bulletted lists in the same section.